plugins/packs-integrations.js

```diff
@@ -415,15 +423,24 @@ async function pluginPacksAndIntegrationsData(context, options) {
           const registryPackData = [];
           for (const registry of packData.spec.registries) {
             const url = `${packUrl}${packData.spec.name}/registries/${registry.uid}?cloudType=${cloudType}&layer=${packData.spec.layer}`;
-            registryPackData.push(callRateLimitAPI(() => api.get(url)));
+            registryPackData.push(
+              callRateLimitAPI(() => {
+                return api.get(url);
+              })
+            );
           }
           return registryPackData;
         });
         const flatted = promisesPackDetails.flat();
         const results = await Promise.allSettled(flatted);
-        apiPacksData = results
-          .filter((result) => result.status === "fulfilled" && result.value?.data)
-          .map((result) => result.value.data);
```

Comment:
Nothing wrong with this code. It's just hard to read. So for future debugging, the code is updated to be more readable.